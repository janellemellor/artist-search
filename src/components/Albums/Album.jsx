import React from 'react';
import PropTypes from 'prop-types';
import { Link } from 'react-router-dom';

const Album = ({title,}) => (
  <section>
    <h3>{title}</h3>
    <
  </section>
);

Album.propTypes = {
  title: PropTypes.string.isRequired,
  id: PropTypes.string.isRequired,
  coverArt: PropTypes.string.isRequired,
};

export default Album;
