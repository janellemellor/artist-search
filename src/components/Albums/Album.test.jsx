import React from 'react';
import { shallow } from 'enzyme';
import Album from './Album.jsx';

describe('Album Component', () => {
  it('matches a snapshot', () => {
    const wrapper = shallow(<Album title="Opiate" id="002fe68d-3414-4925-a9a6-bc9bc6336b06" coverArt="https://placedog.net/640/480?random" />);
    expect(wrapper).toMatchSnapshot();
  });
});

