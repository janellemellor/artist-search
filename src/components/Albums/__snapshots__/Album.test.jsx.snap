// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Album Component matches a snapshot 1`] = `
<section>
  <Link
    to="/Opiate/002fe68d-3414-4925-a9a6-bc9bc6336b06"
  >
    <h3>
      Opiate
    </h3>
    <img
      alt="Opiate"
      src="https://placedog.net/640/480?random"
    />
  </Link>
</section>
`;
